### PR TITLE
Matt/bitcoin instructions

### PR DESCRIPTION
**Pull request by tirwyd-sarmes**
A basic set of instructions for %btc-wallet pending a docs section for urbit apps.

### CODE REVIEW

File: content/Bitcoin-wallet.md [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+title = "Bitcoin Wallet"
 +++
 
 
 
 The latest Urbit release includes a native Bitcoin wallet. Please read the following security recommendations and disclaimers carefully. Urbit.org cannot recover any lost keys.
 
+### Instructions
 
+Run `|start %btc-wallet` in the dojo. In the first row of tiles on Landscape you should now see an orange bitcoin tile. Clicking on it will prompt a two-step process:
```
Comment (jalehman) [inline content/Bitcoin-wallet.md:11]: Does the page need to be refreshed?

File: content/Bitcoin-wallet.md [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+
+1. Enter a provider who is connected to a full-node (such as `~matwet`). This provider will maintain a synced Bitcoin ledger and process your transactions.
```
Comment (jalehman) [inline content/Bitcoin-wallet.md]: And also index your addresses for balance determination.

File: content/Bitcoin-wallet.md [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
+
+2. The next step is to provide an xPub key. An xPub key will allow %btc-wallet to automatically generate new public addresses when bitcoin is sent to your ship. To provide an xPub you can either submit your master ticket (which can be downloaded from [Bridge](https://bridge.urbit.org)) or directly import your xPub key (such as from a hardware wallet like [Ledger](https://support.ledger.com/hc/en-us/articles/360011069619-Extended-public-key) or [Trezor](https://wiki.trezor.io/User_manual:Displaying_account_public_key_(XPUB))).
```
Comment (jalehman) [inline content/Bitcoin-wallet.md]:
```suggestion
2. The next step is to provide an extended public key (or xpub). Providing your xpub allows the Bitcoin Wallet to automatically generate new public addresses (called *child public keys*—you can learn more about this [here](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)) when bitcoin is sent to your ship. To provide an xpub you can either enter your master ticket (which can be downloaded from [Bridge](https://bridge.urbit.org)), or directly import your xpub (such as from a hardware wallet like [Ledger](https://support.ledger.com/hc/en-us/articles/360011069619-Extended-public-key) or [Trezor](https://wiki.trezor.io/User_manual:Displaying_account_public_key_(XPUB))).
```

File: content/Bitcoin-wallet.md [hunk 4]
```diff
@@ -1,12 +1,22 @@
...
+
+Using the landscape tile you can now view your balance and send bitcoin. All that is required to send bitcoin is the recipient’s ship name.
```
Comment (jalehman) [inline content/Bitcoin-wallet.md]:
```suggestion
Clicking on the Landscape tile will take you to the Bitcoin Wallet application, where you can view your balance and send bitcoin. All that you'll need to do is enter the recipient's Urbit ID (if they have configured the wallet using the above steps) or a Bitcoin address.
```

File: content/Bitcoin-wallet.md [hunk 5]
```diff
@@ -1,12 +1,22 @@
...
+
+If you are interested in [being a provider](https://subject.network/posts/btc-wallet-config/#connecting-a-provider-to-a-full-node) or [running a bitcoin full node](https://subject.network/posts/pi-fullnode-urbit/#bitcoind) on the Urbit network please check out ~sitful-hatred’s excellent tutorials.
```
Comment (jalehman) [inline content/Bitcoin-wallet.md]:
```suggestion
If you are interested in [being a provider](https://subject.network/posts/btc-wallet-config/#connecting-a-provider-to-a-full-node) or [running a bitcoin full node](https://subject.network/posts/pi-fullnode-urbit/#bitcoind) on the Urbit network please check out `~sitful-hatred`’s excellent tutorials.
```

**Review by jalehman — CHANGES_REQUESTED** [review]
A few small changes, and then looks good!

File: content/Bitcoin-wallet.md (hunk 2 above)
Comment (jalehman) [reply content/Bitcoin-wallet.md]:
```suggestion
1. Enter a provider who is connected to a full-node (such as `~matwet`). The provider will maintain a synced Bitcoin ledger, index your public addresses to determine your balance, and process your transactions.
```

File: content/Bitcoin-wallet.md (hunk 1 above)
Comment (tirwyd-sarmes) [reply content/Bitcoin-wallet.md:11]: Yes! Good catch.

**Pull request merged**